lua/paramset.lua

```diff
@@ -16,19 +16,22 @@ local ParamSet = {
   tCONTROL = 3,
   tFILE = 4,
   tTAPER = 5,
-  tTRIGGER = 6
+  tTRIGGER = 6,
+  sets = {}
 }
 
 ParamSet.__index = ParamSet
 
 --- constructor
 -- @param name
-function ParamSet.new(name)
+function ParamSet.new(id, name)
   local ps = setmetatable({}, ParamSet)
+  ps.id = id or ""
   ps.name = name or ""
```

Comment:
would this make sense to inherit `id` if `name` not specified?

ie

```
ps.name = name or id or ""
```

(does chaining `or`'s work in lua?)

Reply:
i am keeping this separated like with param ids, in case we have spaces and similar characters in the name (which is currently not even used, but this PR sets it for `mix` pset to keep its params in a separate osc namescpae).